### DISCUSSION TITLE
Want to use already existing domain certificate

### DISCUSSION THREAD

**Discussion by cnavarro1499** [mcguirepr89/BirdNET-Pi#588] [Q&A]
Hi, all

I've been a birdwatcher for years and you've provided me a new obsession!

My BirdNET Pi machine is chugging along happily and is available on my local network, but I kind of wanted to share! I have my own domain with my external IP obfuscated by Cloudflare. I set up my A record to point to birdnet.example.com (obviously that's not it), but am getting an SSL 526 error. I suspect this is due to the certificate locally on the pi not matching my domain. My already existing certificate (which exists on my reverse proxy server) is what Cloudflare would be looking for. I'd rather use that one... Is there a way to configure Caddy to work with my reverse proxy server? Or am I better off requesting a URL from Tim?

Sincerely,

Christine

**Comment by ehpersonal38** [comment]
Hi @cnavarro1499!

Glad to hear you've been enjoying BirdNET-Pi!

I believe someone earlier had the same issue as this, and their solution can be found here: https://github.com/mcguirepr89/BirdNET-Pi/discussions/537#discussioncomment-3609349

Let me know if that works for you or if there's some nuance of your issue that wasn't covered there

**Reply by cnavarro1499** [reply]
Maybe I didn't explain it well, but that is exactly the opposite of what I am trying to do. I'm trying to use the Let's Encrypt cert on another machine, not point to one locally.

**Reply by ehpersonal38** [reply]
Sorry about that, this stuff is a bit over my head admittedly. @mcguirepr89 is a wizard and would know the solution, hopefully he sees this soon. Best of luck
